pages/index.tsx

```diff
@@ -1,64 +1,28 @@
 import Head from 'next/head'
 import Image from 'next/image'
 
-import styles from '@/pages/index.module.css'
+import Header from '@/components/header/header';
+import SearchBar from '@/components/searchBar/searchBar';
+import SearchResults from '@/components/searchResults/searchResults';
+
+import styles from '@/pages/index.module.scss'
 
 export default function Home() {
   return (
     <div className={styles.container}>
       <Head>
-        <title>Create Next App</title>
+        <title>Recipe Calculator App</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
       <main>
-        <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
-
-        <p className={styles.description}>
-          Get started by editing <code>pages/index.js</code>
-        </p>
-
-        <div className={styles.grid}>
-          <a href="https://nextjs.org/docs" className={styles.card}>
-            <h3>Documentation &rarr;</h3>
-            <p>Find in-depth information about Next.js features and API.</p>
-          </a>
-
-          <a href="https://nextjs.org/learn" className={styles.card}>
-            <h3>Learn &rarr;</h3>
-            <p>Learn about Next.js in an interactive course with quizzes!</p>
-          </a>
-
-          <a
-            href="https://github.com/vercel/next.js/tree/canary/examples"
-            className={styles.card}
-          >
-            <h3>Examples &rarr;</h3>
-            <p>Discover and deploy boilerplate example Next.js projects.</p>
-          </a>
-
-          <a href="https://vercel.com/new" className={styles.card}>
-            <h3>Deploy &rarr;</h3>
-            <p>
-              Instantly deploy your Next.js site to a public URL with Vercel.
-            </p>
-          </a>
-        </div>
+        <Header/>
+        <SearchBar/>
+        <SearchResults/>
```

Comment:
I notice these 2 components live in the components directory. A typical pattern is that the components directory house all global components that are used throughout the application, for example a Button. 

Will these 2 components be re usable accross the app? if not then they should be co located within the page that consumes them. for example;

pages/Home/SearchBar/index.tsx
pages/HomeSearchResults/index.tsx

it also sounds like search results could possibly be a separate page? but absolutely fine if you want to render the output on the homepage it can be colocated to the homepage